### PR TITLE
Move generator to separate package

### PR DESCRIPTION
**Pull request by pavelgruba**

### CODE REVIEW

File: packages/devextreme-react/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "devexpress"
+  ],
+  "license": "MIT",
+  "peerDependencies": {
+    "devextreme": "~20.2.4",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0"
+  },
+  "dependencies": {
+    "devextreme-react-generator": "^20.2.4",
```
Comment (IlyaKhD) [inline packages/devextreme-react/package.json:30]: This is a devDependency

File: packages/devextreme-react-generator/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "author": {
+    "name": "Developer Express Inc."
+  },
+  "name": "devextreme-react-generator",
+  "version": "20.2.4",
```
Comment (IlyaKhD) [inline packages/devextreme-react-generator/package.json:6]:
```suggestion
  "version": "1.0.0",
```

Comment (IlyaKhD) [reply packages/devextreme-react-generator/package.json:6]: Let's use semver here

**Comment by IlyaKhD** [comment]
I think the correct way is to move the `/example` to the `/packages/` folder, so Lerna will process its references to the `devextreme-react` package
And possibly rename it to Sandbox, how do you think?

**Review by roman-simionov — APPROVED** [review]

**Pull request merged**